packages/account/src/Sections/Security/DeactivateAccount/deactivate-account-reason.jsx

```diff
@@ -132,9 +132,13 @@ class DeactivateAccountReason extends React.Component {
             } else {
                 this.setState({ remaining_characters: character_limit_no });
             }
-            const regex_rule = `^[0-9A-Za-z .,'-]{0,${max_characters}}$`;
-            if (!new RegExp(regex_rule).test(final_value)) {
-                error.characters_limits = `please insert up to ${max_input_characters_can_use} characters combine both fields.`;
+
+            if (!/^[0-9A-z .,'-]*$/.test(final_value)) {
+                error.characters_limits = localize("Must be numbers, letters, and special characters . , ' -");
+            }
+
+            if (final_value.length > max_characters) {
+                error.characters_count_exceed = `please insert up to ${max_input_characters_can_use} characters combine both fields.`;
```

Comment:
You may need to consult this with copywriting team since number of characters even includes the checkbox characters! so by right it is more than just the 2 fields.

Reply:
`max_input_characters_can_use` doesn't include  checkbox characters, in line 121 they have been excluded already

Reply:
Exactly, which means `final_value` is the result of `otherTradingPlatforms` and 'doToImprove' along with `selected_reasons` characters hence the error message is not correct.